cli/ballerina-packerina/src/main/java/org/ballerinalang/packerina/BuilderUtils.java

```diff
@@ -64,7 +64,7 @@ public static void compileWithTestsAndWrite(Path sourceRootPath,
 
         if (skiptests) {
             outStream.println();
-            compiler.write(bLangPackage, packagePath);
+            compiler.write(bLangPackage, targetPath);
```

Comment:
Why we have changed this to targetPath?

Reply:
We should write the balx to the target path given by the user. If the target path is not given then by default we write the balx inside the target folder.